metadata-service/auth-impl/src/main/java/com/datahub/authentication/token/StatefulTokenService.java

```diff
@@ -179,6 +179,14 @@ public void revokeAccessToken(@Nonnull String hashedToken) throws TokenException
     throw new TokenException("Access token no longer exists");
   }
 
+  public boolean isTokenRevoked(@Nonnull String hashToken) {
+    try {
+      return _revokedTokenCache.get(hashToken);
+    } catch (ExecutionException e) {
```

Comment:
Why would ExecutionException be thrown? Is it worth logging a warn / error?

Reply:
It's the API from Guava's Cache. I wanted a simpler API, to see if a token was revoked or not.